app/controllers/projects_controller.rb

```diff
@@ -32,7 +32,7 @@ def create
         project_path(@project)
       )
     else
-
+      flash[:error] = @project.errors.full_messages.first
```

Comment:
This has the same behaviour as the comment above, so please remove the flash here.

Reply:
Updated in 2365691109fea5b67fd2bc7040892d5ad22edff2